### DISCUSSION TITLE
Days in brew time

### DISCUSSION THREAD

**Discussion by RehmatSChawla** [graphefruit/Beanconqueror#752] [Q&A] [answered]
Is it possible to count days in brew time? It is useful for some of my cold brews. Currently the maximum brew time seems to be 23h 59m 59s (and then ms).
 I can vaguely remember seeing an option to include days in brew time when I first installed the app. I cannot find it anymore, but possibly I'm just mistaken.

**Comment by graphefruit** [comment, marked as answer]
Hello @RehmatSChawla,
thanks for the discussion here.
I don't see a good solution to add "Days" in a brew time, just maybe for cold-brew.
Also I think when adding days, the logic of my code would break, because actually its just supports 23h 59m 59s + ms.
The days where never supported, just the possibility to change the displaying date-format.

**Reply by RehmatSChawla** [reply]
Alright, thanks! I'll make do by leaving notes with the full brew time instead.
